### PR TITLE
Expose Pipeline Render Textures

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 Expose accessing pipeline render textures to user (camera color and depth).
 A follow up PR should expose other pipeline textures (gbuffer, shadow maps, motion vectors etc)

 Design doc:  to user land: https://docs.google.com/document/d/1qBEyak13OS1CdBmRn6brU7izz63ky43-c-7IZgBuJ3s/

### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs [hunk 1]
```diff
@@ -204,7 +204,8 @@ internal static void SetStereoViewAndProjectionMatrices(CommandBuffer cmd, Matri
             }
             else
             {
-                cmd.SetRenderTarget(destination, colorLoadAction, colorStoreAction, depthLoadAction, depthStoreAction);
+                ScriptableRenderer.SetRenderTarget(cmd, destination, colorLoadAction, colorStoreAction, ClearFlag.None, Color.black);
+                //cmd.SetRenderTarget(destination, colorLoadAction, colorStoreAction, depthLoadAction, depthStoreAction);
```
Comment (pbbastian) [inline com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs:208]: Should this still be here?

**Comment by phi-lira** [comment]
Converted to draft as this requires still more work.

File: com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs (hunk 1 above)
Comment (phi-lira) [reply com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs:208]: converted this to Draft PR as it needs more work.

**Comment by phi-lira** [comment]
Closing due to inactivity

**Pull request closed without merging**